spec/requests/home_request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```

Comment:
add controllers specs instead of requests

Reply:
Done, but aren't controllers specs deprecated?